tests/feature.py

```diff
@@ -0,0 +1,8 @@
+"""
+Enumerate all the capabilities in the drivers
+"""
+from enum import Enum
+
+
+class Feature(Enum):
+    AUTHORIZATION_EXPIRED_TREATMENT = 'AutorizationExpiredTreament'
```

Comment:
I'd argue that that should go somewhere in `nutkit/protocol/responses.py` or similar, as those stings are sent over the wire, aren't they?

Reply:
The strings are sent over the wire, I could move it to the protocol. But this one is kind of grey area between the protocol and the test context.

Reply:
I've seen many tests that do `import nutkit.protocol as types`. So it wouldn't be the first place where tests know about the protocol's classes.